tests/test_node_differ.py

```diff
@@ -332,4 +332,13 @@ def test_insert_cjk_punctuations():
     cjk_curr_wikitext = cjk_prev_wikitext.replace('。','、',1)
     expected_changes = {'Section':{'change':1}, 'Punctuation':{'change':1},'Sentence':{'change':1},'Paragraph':{'change':1}}
     diff = EditTypes(cjk_prev_wikitext, cjk_curr_wikitext, lang='ja').get_diff()
+    assert expected_changes == diff
+
+def test_hyphen_words():
```

Comment:
if you want to keep this test case (i'm fine either way), let's move it up so it's next to `test_text_change` given how similar they are